e2e2/test/images/bert-inference/Dockerfile

```diff
@@ -4,20 +4,58 @@ FROM nvidia/cuda:12.5.0-devel-ubuntu22.04
 # Set environment variable to disable interactive prompts
 ENV DEBIAN_FRONTEND=noninteractive
 
-# Install Python 3.11
-RUN apt-get update && apt-get install -y \
-    software-properties-common && \
-    add-apt-repository ppa:deadsnakes/ppa && \
-    apt-get update && \
-    apt-get install -y \
-    python3.11 \
-    python3.11-dev \
-    python3.11-distutils \
-    python3-pip && \
-    rm -rf /var/lib/apt/lists/*
+# Dependency version numbers
+ARG PYTHON=python3.10
+ARG PYTHON_VERSION=3.10.12
+ARG PIP=pip3
 
-# Create a symbolic link to use python3.11 as python
-RUN ln -sf /usr/bin/python3.11 /usr/bin/python
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    curl \
+    emacs \
```

Comment:
if you can trim any of these out I would but nbd 🤔

Reply:
I actually only included those because they were in our other images - https://github.com/aws/aws-k8s-tester/blob/cc66356a28d11fdd4a60573d2a2bbe502a14dbab/e2e2/test/images/neuron/Dockerfile#L39C1-L43C10

Was just trying to keep things consistent and start to identify common patterns amongst our images. Maybe these are required in the other image...? If not, I would definitely advocate for us to remove.

Reply:
yea agreed, its not blocking me